community_server/src/Controller/AppRequestsController.php

```diff
@@ -393,6 +393,16 @@ public function listTransactions($page = 1, $count = 25, $orderDirection = 'ASC'
         $this->set('body', $body);       
     }
     
+    public function getDecayStartBlock()
+    {
+        $transactionsTable  = TableRegistry::getTableLocator()->get('Transactions');
+        $decayStartBlock = $transactionsTable->find()->where(['transaction_type_id' => 9]);
+        if(!$decayStartBlock->count()) {
+            return $this->returnJson(['state' => 'error', 'msg' => 'not found']);
+        }
+        return $this->returnJson(['state' => 'success', 'decay_start' => $decayStartBlock->first()->received]);
+    }
```

Comment:
The docu for this call is missing. Same applies to the changed call returning flag for decay start. If possible please include this in the API docu <3